ale/base/data_naif.py

```diff
@@ -426,6 +439,39 @@ def detector_center_line(self):
         """
         return float(spice.gdpool('INS{}_BORESIGHT_LINE'.format(self.ikid), 0, 1)[0])
 
+    @property
+    def swap_observer_target(self):
+        """
+        Returns if the observer and target should be swapped when determining the
+        sensor state relative to the target. This is defined by a keyword in
+        ISIS IAKs. If the keyword is not defined in any loaded kernels then False
+        is returned.
+
+        Expects ikid to be defined. This should be an integer containing the
+        Naif Id code of the instrument.
+        """
+        try:
+            swap = spice.gcpool('INS{}_SWAP_OBSERVER_TARGET'.format(self.ikid), 0, 1)[0]
+            return swap.upper() == "TRUE"
+        except:
+            return False
+
+    @property
+    def correct_lt_to_surface(self):
+        """
+        Returns if light time correction should be made to the surface instead of
+        to the center of the body. This is defined by a keyword in ISIS IAKs.
+        If the keyword is not defined in any loaded kernels then False is returned.
+
+        Expects ikid to be defined. This should be an integer containing the
+        Naif Id code of the instrument.
+        """
+        try:
+            surface_correct = spice.gcpool('INS{}_LT_SURFACE_CORRECT'.format(self.ikid), 0, 1)[0]
```

Comment:
Do we expect this key in IKs or IAKs?

Reply:
We need to be able to use IAKs because ISIS depends on IAKs right now.

Reply:
It seems fine for now for the sake fo a result, but I think after we get a clean result from ALE + ISIS it's worth evaluating how to get ISIS specific stuff out of ALE and back into ISIS.